metadeploy/api/flows.py

```diff
@@ -59,6 +59,15 @@ def pre_flow(self, coordinator):
         return self.logger
 
     def post_flow(self, coordinator):
+        """
+        Send a password reset email when completing scratch org JobFlows
+        """
+        org_config = coordinator.org_config
+        if org_config.config.get("scratch"):
+            org_config.salesforce_client.restful(
```

Comment:
This call to the endpoint seems to run fine but I never got an email when testing locally (I did get an email for the Plan installing successfully)

Reply:
Hmm, I'm not sure why it didn't work. I just tried again with a fresh scratch org that had never had a password set, and I got the email. Are you sure the email for the org's admin user got set to yours when creating the ScratchOrgInfo in the dev hub? If it did not, it would default to the email of the Dev Hub user, which is probably Jonny's or Kit's...

Reply:
I wouldn't spend too much time on this if it just doesn't work. Triggering the password reset is not a very important feature now that we have a button for accessing the org.

Reply:
This call itself is working for me when I make sure this block is being executed (I'm getting the pwd-reset email fine), but... even on a newly-created scratch org (I'm using the EDA installer), the `org_config` here does not have a `scratch` key.

The org config we're storing in the DB for a scratch org has keys:

```
[
  "days",
  "email",
  "org_id",
  "created",
  "scratch",
  "username",
  "is_sandbox",
  "namespaced",
  "sfdx_alias",
  "config_file",
  "config_name",
  "access_token",
  "date_created",
  "instance_url",
  "set_password",
  "refresh_token"
]
```

But the org_config we have access to in `post_flow` has keys:

```
[
  "access_token",
  "instance_url",
  "refresh_token",
  "signature",
  "scope",
  "id",
  "token_type",
  "issued_at",
  "userinfo",
  "org_type",
  "is_sandbox",
  "instance_name"
]
```

So I'm not sure if there's a better way we can determine which orgs to trigger this logic on?

Reply:
Decided to check for the existence of a local ScratchOrg instead (continued on the other thread)